matrix_client/api.py

```diff
@@ -97,20 +97,15 @@ def validate_certificate(self, valid):
         self.validate_cert = valid
         return
 
-    def register(self, login_type, **kwargs):
+    def register(self, content={}, query_params={}):
         """Performs /register.
 
         Args:
-            login_type(str): The value for the 'type' key.
-            **kwargs: Additional key/values to add to the JSON submitted.
+            content(dict): The request payload. Should include "type" such as "m.login.password" for all non-guest registrations.
```

Comment:
Could you make all required keys listed on the line below e.g. 
```
content(dict): The request payload.
               'auth.type' : Must be included with a value such as "m.login.password"
               '...': blah
               See the spec for additional accepted options.
```

(I tried to find any sources on the accepted way to show required keys for a dict in a docstring but found nothing, so if you have any insights on this then follow that :laughing: )